app/views/parks/show.html.erb

```diff
@@ -44,6 +44,31 @@
     <% if !review.id.nil?%>
           <li class="inline-list">
             <%= review.title%>
+            <% flag = false %>
+
+            <% if !review.votes.nil?  %>
+              <% review.votes.each do |vote|%>
+                <% if vote.user_id == current_user.id %>
+                  <% flag = true %>
+                <% end %>
+              <% end %>
+            <% end %>
+
+            <% if flag %>
+
+                <%= button_to 'Up', {controller: "votes", action: "update", park_id: @park.id, review_id: review.id, vote: "up"} , method: :patch  %>
```

Comment:
I've changed your `routes.rb` check that out first. Change this to use path helpers `vote_path(review, review.votes.find_by(user: current_user))`